input/input.go

```diff
@@ -107,6 +115,23 @@ func NewBaseInput(outpoint *wire.OutPoint, witnessType WitnessType,
 	return &input
 }
 
+// NewCsvBaseInput assembles a new csv-locked input that can be used to
+// construct a sweep transaction.
+func NewCsvBaseInput(outpoint *wire.OutPoint, witnessType WitnessType,
```

Comment:
nit: naming suggestion `NewCSVInput`, idt we need base in the name

Reply:
renamed